include/oneapi/dpl/pstl/execution_impl.h

```diff
@@ -105,25 +78,163 @@ template <typename _ExecutionPolicy>
 using __allow_parallel = typename __internal::__policy_traits<::std::decay_t<_ExecutionPolicy>>::__allow_parallel;
 
 template <typename _ExecutionPolicy, typename... _IteratorTypes>
-auto
-__is_vectorization_preferred(_ExecutionPolicy& __exec)
-    -> decltype(__internal::__lazy_and(__exec.__allow_vector(),
-                                       __internal::__is_random_access_iterator_t<_IteratorTypes...>()))
+constexpr auto
+__is_vectorization_preferred()
 {
-    return __internal::__lazy_and(__exec.__allow_vector(),
-                                  __internal::__is_random_access_iterator_t<_IteratorTypes...>());
+    return ::std::conjunction<decltype(::std::decay_t<_ExecutionPolicy>::__allow_vector()),
+                              __internal::__is_random_access_iterator_t<_IteratorTypes...>>();
 }
 
 template <typename _ExecutionPolicy, typename... _IteratorTypes>
-auto
-__is_parallelization_preferred(_ExecutionPolicy& __exec)
-    -> decltype(__internal::__lazy_and(__exec.__allow_parallel(),
-                                       __internal::__is_random_access_iterator_t<_IteratorTypes...>()))
+constexpr auto
+__is_parallelization_preferred()
```

Comment:
It seems `__is_parallelization_preferred()` is not used anymore.

Reply:
It's not true.

Reply:
Find shows me just one result.
Please point me file and number of line where we have usage of it.

Reply:
`__for_loop_repack` -> `__for_loop_impl` -> `using _IsVector = decltype(oneapi::dpl::__internal::__use_vectorization<_ExecutionPolicy, _Ip>());` -> `return __use_par_vec_helper<_Ip>::template __use_vector<_ExecutionPolicy>();` -> `struct __use_par_vec_helper` :

```C++
template <typename _Ip>
struct __use_par_vec_helper<_Ip, ::std::enable_if_t<::std::is_integral_v<_Ip>>>
{
    template <typename _ExecutionPolicy>
    static constexpr auto
    __use_vector()
    {
        return ::std::decay_t<_ExecutionPolicy>::__allow_vector();
    }

    template <typename _ExecutionPolicy>
    static constexpr auto
    __use_parallel()
    {
        return ::std::decay_t<_ExecutionPolicy>::__allow_parallel();
    }
};
```

Reply:
I am sorry, but I cannot find "__is_parallelization_preferred()" call in the code which you mentioned above.